content/applications/hr/employees/new_employee.rst

```diff
@@ -231,36 +249,38 @@ This tab provides various fields for different information, depending on the cou
 located. Different fields are configured for different locations, however some sections appear
 regardless.
 
-- :guilabel:`Status`: If applicable, select a :guilabel:`Related User`, :guilabel:`Job Position`,
-  and :guilabel:`Language` with the drop-down menus. Type in the :guilabel:`Registration Number of
-  the Employee`, and the :guilabel:`NIF Country Code` if available.
-- :guilabel:`Fleet`: If the employee has access to a company car, enter the :guilabel:`Mobility
-  Card` information here.
-- :guilabel:`Timesheets`: Enter the employee's cost per hour in a $XX.XX format. This is factored in
-  when the employee is working at a work center. This value affects the employee's pay, and may also
-  affect manufacturing costs for a product, if the value of the manufactured product is not a fixed
-  amount.
-- :guilabel:`Attendance`: This section may only appear for Belgian companies. Enter the
-  :guilabel:`INSZ or BIS` number if applicable. :guilabel:`Badge ID` and a :guilabel:`PIN Code` can
-  be entered here, if the employee needs/has one. Click :guilabel:`Generate` next to the
-  :guilabel:`Badge ID` to create a badge ID.
+- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related
+  User`, with the drop-down menus.
+- :guilabel:`Payroll`: If applicable, enter the :guilabel:`Registration Number of the Employee` in
+  this section. The :guilabel:`Attestation (N-1)` and :guilabel:`Attestation (N)` sections appear
+  ony for Belgian companies, and will not be visible for other locations. These sections log the
+  days that will be paid to the new employee. Enter any :guilabel:`Amount to recover`,
+  :guilabel:`Number of days`, and :guilabel:`Recovered Amount` of simple holiday pay, for both N and
+  N-1 categories. For the double holiday pay to recover section, click :guilabel:`Add a line`, and
+  enter the number of :guilabel:`Months`, the :guilabel:`Amount`, and :guilabel:`Occupation Rate`.
+  Repeat for all entries. Click the :guilabel:`🗑️ (trash can)` to delete a line.
+- :guilabel:`Application Settings`: Enter the employee's :guilabel:`Hourly Cost` in a $XX.XX format.
+  This is factored in when the employee is working at a work center. This value affects the
+  manufacturing costs for a product, if the value of the manufactured product is not a fixed amount.
+  If applicable, enter the :guilabel:`Fleet Mobility Card` number.
+- :guilabel:`SD WORX`: Enter the employee's :guilabel:`SDWorx code` in this field, if applicable.
+- :guilabel:`Attendance/Point of Sale/Manufacturing`: The employee's  :guilabel:`PIN Code` and
+  :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
+  :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID.
+- :guilabel:`Payroll`: Select the employee's :guilabel:`Job Position` from the drop-down menu.
```

Comment:
Not sure why there's a second Payroll section in your database/image, but in Runbot everything is consolidated into one Payroll section, which makes a lot more sense than having two!

Reply:
That's odd- both in mt database I made for the tour and app page (started with no data) and the runbot I checked- they both have payroll listed twice (doesn't make sense to me either!) I ept it in the order of the columns as they appear.
![image](https://github.com/odoo/documentation/assets/121518652/7820c805-155c-46e3-9e23-2503f6d407d8)
I will check and see if this is a bug we can have fixed, or if there is a reason this is appearing this way.

Reply:
OK, ticket is submitted! https://www.odoo.com/web#cids=3&menu_id=4720&action=333&active_id=49&model=project.task&view_type=form&id=3422904